addons/content-index/src/main/java/org/commonjava/indy/content/index/IndexingContentManagerDecorator.java

```diff
@@ -257,6 +257,25 @@ private Transfer getIndexedTransfer( final StoreKey storeKey, final StoreKey top
         Logger logger = LoggerFactory.getLogger( getClass() );
         logger.trace( "Looking for indexed path: {} in: {} (entry point: {})", path, storeKey, topKey );
 
+        try
+        {
+            ArtifactStore store = storeDataManager.getArtifactStore( storeKey );
+            if ( store.isDisabled() )
+            {
+                logger.info( "Content not available in index caching layer due to store disabled for {} in group {}",
+                             storeKey, topKey );
+                return null;
+            }
+        }
+        catch ( IndyDataException e )
+        {
+            logger.error(
+                    String.format( "Failed to lookup store: %s (in membership of: %s). Reason: %s", storeKey, topKey,
+                                   e.getMessage() ), e );
+            //FIXME: Should this be null?
```

Comment:
One concern here is if the store is a constituent of a group, and if a workflow ex pop up, will this break the subsequent processing of the group for other constituents?